### PR TITLE
Week 4 

### PR DESCRIPTION
**Pull request by Josofico**
# Finally we can put what we learn in week 4 in our project.

### CODE REVIEW

File: student-blogs/kick-off-2479/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "dayjs": "^1.11.7"
+  },
+  "devDependencies": {
+    "sass": "^1.61.0",
```
Comment (Lealdos) [inline student-blogs/kick-off-2479/package.json:19]: Please add SASS to the dependencies section of your package.json file, as without it, you won't be able to style your application in the way it's currently implemented

File: student-cvs/kick-off-2479/src/styles/base.scss [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+}
+
+body {
+    font-family: "Century Gothic", sans-serif;
+    line-height: 2;
+    color: var(--color-asphalt);
+}
+
+.information {
+    display: block;
```
Comment (Lealdos) [inline student-cvs/kick-off-2479/src/styles/base.scss:61]: This element has already in block property, so this line is not necessary

**Review by Lealdos — CHANGES_REQUESTED** [review]
I see that you have implemented CSS variables using ':root'. I'm not sure if it's a better option than using '$', but if that works for you, it's okay.

**Review by Lealdos — APPROVED** [review]
everything done

**Review by dlozanoc82 — APPROVED** [review]
Good job. You can check my PR https://github.com/undefined-academy/semana-4/pull/84

**Comment by cadifilippo** [comment]
Quita los cambios del gitignore, si lo necesitas puedes ignorar el propio gitignore

**Review by Lealdos — APPROVED** [review]

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,378 @@
+# Created by https://www.toptal.com/developers/gitignore/api/macos,node,nextjs,react,intellij,linux,windows
```
Comment (cadifilippo) [inline .gitignore:1]: No debes agregar el gitignore en el PR

**Review by cadifilippo — CHANGES_REQUESTED** [review]
quitar .gitignore

**Review by Josofico — COMMENTED** [review]
Just deleting gitignore

**Review by Josofico — COMMENTED** [review]
Restarting

**Review by Josofico — COMMENTED** [review]
Done

**Review by Josofico — COMMENTED** [review]
Deleting files.

**Review by Josofico — COMMENTED** [review]
.

**Pull request merged**